eahub/templates/account/password_reset_from_key.html

```diff
@@ -1,16 +1,62 @@
-{% extends 'base.html' %}
+{% extends 'base_bs5.html' %}
 
-{% block title %}Enter new password{% endblock %}
+{% load crispy_forms_field crispy_forms_tags socialaccount static %}
 
+{% block navbar %}
+  {% include 'components/navbar.html' %}
+{% endblock navbar %}
 
+{% block title %}Set a New Password{% endblock %}
 
 {% block content %}
-<div class="container user-action">
-<h1>Set a new password</h1>
-<form method="POST">
-  {% csrf_token %}
-  {{ form.as_p }}
-  <input class="btn btn-default btn-xs" type="submit" value="Change my password">
-</form>
-</div>
+  <div class="container account-floating-form">
+    <h1>Set a New Password</h1>
+
+    {% include 'includes/messages.html' with messages=messages %}
+  
+    <form method="POST">
+      {% csrf_token %}
+    
+      {% if token_fail %}
+        <div class="form-group mb-4">
+          <p class="error">
+            Unfortunately this password reset url isn't valid anymore.
+            Please try to <a href="{% url "account_reset_password" %}">reset your password here</a>,
+            or reach out to the support at contact@eahub.org.
```

Comment:
Maybe rather tech@eahub.org.?

Also, we should set this in environment variables or with django singleton (don't need to do this now)

Reply:
I would recommend reserving tech@ for technical matters only, eg sentry or DNS monitoring.

Ideally I would put there support@eahub.org, but looks like it wasn't set up.

Would that make sense to you?